### DISCUSSION TITLE
Configuration Migration Tool User Mapping and pac data Import

### DISCUSSION THREAD

**Discussion by eleung83** [microsoft/powerplatform-vscode#708] [Ideas]
When using the Configuration Migration Tool to build a data schema for the the import and export of data, one is also allowed to provide a User mapping file during the import process if their data schema contains user records.

Yet, if I attempt to perform the same actions using the pac cli commands to perform the same import and export functions ("pac data export" and "pac data import"), there seems to be no way I can provide this user mapping file, which then renders me unable to auto export and import data from one environment to another which has some dependency on the user entity (for example teams).

Is there any way to somehow force the pac data import commands to accept a user mapping file?

**Comment by petrochuk** [comment]
Converting to discussion so it can be voted on

**Comment by rajyraman** [comment]
@eleung83 - zip file exported by CMT contains both schema and data. So, I am unable to understand why schema file has to be provided separately during import?
